backend/backend.go

```diff
@@ -359,7 +359,8 @@ func (backend *Backend) createAndAddAccount(
 				backend.log.WithError(err).Error("could not find ERC20 token")
 				continue
 			}
-			backend.createAndAddAccount(token, erc20TokenCode, token.Name(), signingConfigurations, nil)
+			erc20AccountCode := fmt.Sprintf("%s-%s", code, erc20TokenCode)
```

Comment:
A grep shows this already in other 3 places. Time to factor out into a helper func?

![image](https://user-images.githubusercontent.com/25405/118475693-d8387000-b70c-11eb-8f64-bd7b259ac09f.png)

Reply:
They all seem to have a different context though, and happen to use the same syntax by coincidence, not because there is something special about %s-%s.

Reply:
Then I would definitely prefix them with that context:

```go
fmt.Sprintf("context-%s-%s", ...)
```

so to not give an impression they are interchangeable